selectors.yml

```diff
@@ -207,6 +207,27 @@ selectors:
   2021:
     selector: 13116810400804392105
     name: "ronin-testnet-saigon"
+  200810:
+    selector: 3789623672476206327
+    name: "bitcoin-testnet-bitlayer-1"
+  808813:
+    selector: 5535534526963509396
+    name: "ethereum-testnet-sepolia-bob-1"
+  37111:
+    selector: 6827576821754315911
+    name: "ethereum-testnet-sepolia-lens-1"
+  1328:
+    selector: 1216300075444106652
+    name: "ethereum-testnet-sepolia-sei-1"
```

Comment:
sei isn't an L2 right, afaik it's a cosmos EVM chain

Reply:
I believe you are right, I got confused about `CosmWasm` and I thought it only rely in that platform for the contracts, not that it was based.
If you are correct, the proper name should be `cosmos-testnet-sei`?

Reply:
changed to the new defined name.